omegaconf/nodes.py

```diff
@@ -115,7 +116,7 @@ def _is_missing(self) -> bool:
                 # Will happen if interpolation is accessing keys that does not exist.
                 return False
         else:
-            ret = self._value() == "???"
+            ret = _is_missing_literal(self._value())
```

Comment:
What about just: `return _is_missing_value(self)`

Reply:
good idea.

Reply:
Not a big deal but I see you didn't exactly take my suggestion, any reason?

Reply:
because I read it too fast :).